src/pages/dataTableUtilities/actions/tableActions.js

```diff
@@ -96,6 +86,17 @@ export const showOverStocked = () => refreshData();
  */
 export const showStockOut = () => refreshData();
 
+/**
+ * Wrapper around showFinalised/showNotFinalised to toggle between. Determines the
+ * correct action to dispatch.
+ *
+ * @param {Bool} showFinalised Indicator wheter finalised rows are currently displayed.
+ */
+export const toggleShowFinalised = showingFinalised => {
+  if (showingFinalised) return showNotFinalised();
```

Comment:
calling `showNotFinalised()` just returns a plain JS object in the form `type: ACTIONS.SHOW_FINALISED` etc.  This is more DRY code rather than chaining of any actions

This is the equivalent to having the method be:

```
export const toggleShowFinalised = showFinalised => {
    if (showFinalised) return {type: ACTIONS.SHOW_FINALISED }
    return { type: ACTIONS.SHOW_NOT_FINALISED {
}
```

Which is essentially the equivalent to what you've suggested, except by using an action creator, there is consistency and less code duplication which are subject to errors.

 Will address the question about conditional in next comment.